movie/src/components/Footer/index.jsx

```diff
@@ -0,0 +1,13 @@
+import "./Footer.css";
+const footerLogo =
+  "	https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg";
+
+const Footer = () => {
+  return (
+    <div>
+      {/* <img src={footerLogo} alt="" /> */}
```

Comment:
why is this commented?

Reply:
Sorry bong, I forgot remove it.

Reply:
It should be in Footer PR, I will remove Footer folder.